fuzzers/libafl/fuzzer.py

```diff
@@ -11,33 +11,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
 """Integration code for AFLplusplus fuzzer."""
 
-# This optimized afl++ variant should always be run together with
-# "aflplusplus" to show the difference - a default configured afl++ vs.
-# a hand-crafted optimized one. afl++ is configured not to enable the good
-# stuff by default to be as close to vanilla afl as possible.
-# But this means that the good stuff is hidden away in this benchmark
-# otherwise.
-
 import os
+import subprocess
 
-from fuzzers.aflplusplus import fuzzer as aflplusplus_fuzzer
+from fuzzers.afl import fuzzer as afl_fuzzer
+from fuzzers import utils
 
 
 def build():  # pylint: disable=too-many-branches,too-many-statements
     """Build benchmark."""
-    aflplusplus_fuzzer.build("tracepc")
+    afl_fuzzer.prepare_build_environment()
+    os.environ['CC'] = '/libafl/fuzzers/fuzzbench/target/release/libafl_cc'
+    os.environ['CXX'] = '/libafl/fuzzers/fuzzbench/target/release/libafl_cxx'
+    os.environ['FUZZER_LIB'] = '/libAFLDriver.a'
+    utils.build_benchmark()
 
 
 def fuzz(input_corpus, output_corpus, target_binary):
     """Run fuzzer."""
-    run_options = ['-l', '1']
-
-    os.environ['AFL_FAST_CAL'] = '1'
-    os.environ['AFL_CMPLOG_ONLY_NEW'] = '1'
-
-    aflplusplus_fuzzer.fuzz(input_corpus,
-                            output_corpus,
-                            target_binary,
-                            flags=(run_options))
+    afl_fuzzer.prepare_fuzz_environment(input_corpus)
+    dictionary_path = utils.get_dictionary_path(target_binary)
+    command = ([target_binary])
```

Comment:
nit: paren not needed.

Reply:
just removed libafl, it needs some more rework and dominik is now gone for the weekend.